src/main.py

```diff
@@ -1,33 +1,53 @@
 #!/usr/bin/env python3
 import asyncio
 import functools
+import logging
 
+import libusb_package
 import odrive
+import usb.util
 from nicegui import app, ui
 
 from controls import controls
 
+logging.getLogger("nicegui").setLevel(logging.ERROR)
```

Comment:
this will not do much yet, but it will once https://github.com/zauberzeug/nicegui/pull/1497 is released